LaTeX/LaTeX.sublime-syntax

```diff
@@ -663,7 +663,7 @@ contexts:
           (\\)
           (?:
             [cC]ite(?:author|title|year|date|url|t|p|alt|alp|text|yearpar)?
-          | (?:[pP]aren|foot|[tT]ext|[sS]mart|super|[aA]|no|full|footfull)cite|footcitetext
+          | (?:[pP]aren|foot|[tT]ext|[sS]mart|super|[aA]|no|full|footfull)cite(?:s)?|footcitetext
```

Comment:
This match seems to broad to me. I'm not an expert, but do `\footcites` or `\nosites` exist?

If the `s` ist just for parencizes, add it as a separate alternative. Or, if parencites is the only command that has this weird mix of optional arguments, it might even make more sense to have it in its own match and push its own set of patterns compared to all of these.

cc @randy3k